src/dbnode/persist/fs/commitlog/files.go

```diff
@@ -38,6 +93,13 @@ type File struct {
 	Index    int64
 }
 
+func newErrorWithPath(err error, path string) ErrorWithPath {
```

Comment:
same for this ctor, move it next to the associated type at the bottom of the file